### DISCUSSION TITLE
widget.styles.height = "initial"

### DISCUSSION THREAD

**Discussion by mattthhh** [Textualize/textual#4378] [Q&A]
Hello, is there a way to reset the value of a CSS attribute of a widget as initial in the python code ?

**Comment by TomJGooding** [comment]
You can't set a style value to "initial" programmatically with Python, as you've probably already discovered this will crash as this expects a value that can be parsed as a height, etc.

I'm struggling to find a way of doing this without diving deep into the internals. Honestly it might be best just to reset using the value you'll find in the `DEFAULT_CSS` for that widget (or parent class).

**Reply by mattthhh** [reply]
Yes, it's a quite good idea to get it by `DEFAULT_CSS`, however I need to parse the variable... Anyway I found a workaround with default value in CSS Style file and affecting value in the source code. Thanks ! :)
